setup.py

```diff
@@ -21,7 +21,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     license="AGPLv3+",
-    install_requires=["SQLALchemy", "alembic", "securedrop-sdk", "python-dateutil", "arrow"],
+    install_requires=["SQLAlchemy", "alembic", "securedrop-sdk", "python-dateutil", "arrow"],
```

Comment:
Scary, but this appears to be the brand...

Reply:
Not scary in this case. While `pip` is case-sensitive, the Python package names are not. ([PEP 426] via [source1] and [source2])

Good to fix it, though. :+1: 

  [PEP 426]: https://peps.python.org/pep-0426/#name
  [source1]: https://stackoverflow.com/a/31585483
  [source2]: https://stackoverflow.com/a/72563527